opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -517,7 +518,7 @@ class Well {
     bool updateWellGuideRate(bool available, double guide_rate, GuideRateTarget guide_phase, double scale_factor);
     bool updateWellGuideRate(double guide_rate);
     bool updateAvailableForGroupControl(bool available);
-    bool updateEfficiencyFactor(double efficiency_factor);
+    bool updateEfficiencyFactor(double efficiency_factor, bool se_efficiency_in_network = true);
```

Comment:
The default value for `bool se_efficiency_in_network = true` is not very useful, since the only usage of the function is for parsing keyword WEFAC, the default behavoir is specified through the definition of the keyword WEFAC.

Reply:
Sure, updated a test instead.